example/generate_from_openapi/pub_tavern.py

```diff
@@ -0,0 +1,39 @@
+import yaml
+from coreapi import Client
+import openapi_codec
+import sys
+
+
+def generate_tavern_yaml(json_path):
+    client = Client()
+    d = client.get(json_path, format="openapi")
+
+    test_dict = {}
+    for test_name in d.links.keys():
+        test_dict["test_name"] = test_name
+
+        request = {
+            "url": d.links[test_name].url,
+            "method": str.upper(d.links[test_name].action),
+        }
+
+        response = {"strict": False, "status_code": 200}
+        inner_dict = {"name": test_name, "request": request, "response": response}
+
+        test_dict["stages"] = [inner_dict]
+        print(test_dict)
+        print(yaml.dump(test_dict, explicit_start=True, default_flow_style=False))
+
+
+def display_help():
+    print("pub_tavern.py <url to openapi.json>")
+    print(
+        "eg: pub_tavern.py https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/json/petstore-simple.json"
```

Comment:
I tested the other example files and noticed the d.links dictionary can be empty (so the for loop doesn't run)

the particular cases are
- https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/json/petstore.json
- https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/json/uber.json

I put together a possible fix. @lingster , I'll submit a pull request into your branch. Feel free to merge if it looks okay -- that should automatically update this pull request.

Reply:
https://github.com/lingster/tavern/pull/1